# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/gc/Zone.h
# Commit: b1117df3b91d
# Full Hash: b1117df3b91d10d68d10b399637cf9ac295057e6
# Author: Jon Coppeard <jcoppeard@mozilla.com>
# Date: 2019-11-01 16:13:11
# Regressor Bug: 1587096
# File Overlap Count: 1
# Description:
#   Bug 1587096 - Part 3: Implmement FinalizationGroup r=sfink
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D49946
# ==============================================================================

diff -r cefc0474e13c -r b1117df3b91d js/src/gc/Zone.h
--- a/js/src/gc/Zone.h	Fri Nov 01 10:37:40 2019 +0000
+++ b/js/src/gc/Zone.h	Fri Nov 01 10:37:47 2019 +0000
@@ -631,6 +631,8 @@
   // a non-zero value since bug 1458011.
   uint32_t detachedTypedObjects = 0;
 
+  void finishRoots();
+
  private:
   // A map from finalization group targets to a list of finalization records
   // representing groups that the target is registered with and their associated